mobile/test/java/org/chromium/net/impl/ErrorsTest.java

```diff
@@ -0,0 +1,109 @@
+package org.chromium.net.impl;
+
+import static org.chromium.net.impl.Errors.mapEnvoyMobileErrorToNetError;
+import static org.junit.Assert.assertEquals;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.SmallTest;
+import io.envoyproxy.envoymobile.engine.EnvoyFinalStreamIntelImpl;
+import io.envoyproxy.envoymobile.engine.UpstreamHttpProtocol;
+import org.junit.rules.ExpectedException;
+import org.chromium.net.impl.Errors.NetError;
+import org.junit.runner.RunWith;
+import org.junit.Rule;
+import org.junit.Test;
+
+@RunWith(AndroidJUnit4.class)
+public class ErrorsTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  @SmallTest
```

Comment:
I don't think you need `@SmallTest` here and throughout this file.

Reply:
Cargo culting, removed.  I see other tests use this annotation - is it useless?  If so, should it be removed in those tests too?  e.g. https://github.com/envoyproxy/envoy/blob/d471d4882a9a484d52fc44735c55f5d96ee85820/mobile/test/java/org/chromium/net/BrotliTest.java#L49

Reply:
Based on the docs: https://developer.android.com/reference/androidx/test/filters/SmallTest, it's not clear to me why we introduced this annotation in our tests.  Do the JUnit test runners on Android use it to schedule the tests?

Reply:
I believe the default is `@SmallTest` and typically we annotate it at the class level, but it's fine to keep it, too.